pipeline/metadata/flatten_base.py

```diff
@@ -20,6 +23,92 @@
 ]
 
 
+def get_common_name(cert_name: x509.Name) -> Optional[str]:
+  """Get the Common Name of a certificate subject or issuer.
+
+  Args:
+    cert_name: x509.Name representing a certificate subject or issuer
+
+  Returns:
+    Common Name as a string
+  """
+  try:
+    attributes = cert_name.get_attributes_for_oid(x509.oid.NameOID.COMMON_NAME)
+    if attributes:
+      return attributes[0].value
+  except x509.AttributeNotFound:
+    logging.warning('x509.AttributeNotFound: Common Name\n')
+  return None
+
+
+def get_alternative_names(cert: x509.Certificate) -> List[str]:
+  """Get the Subject Alternative Names of a certificate.
+
+  Args:
+    cert: x509.Certificate containing parsed certificate fields
+
+  Returns:
+    list of alternative names
+  """
+  try:
+    ext = cert.extensions.get_extension_for_oid(
+        x509.ExtensionOID.SUBJECT_ALTERNATIVE_NAME)
+    # Cast to x509.SubjectAlternativeName to avoid mypy error.
+    san_ext = cast(x509.SubjectAlternativeName, ext.value)
+    return san_ext.get_values_for_type(x509.DNSName)
+  except x509.extensions.ExtensionNotFound:
+    logging.warning(
```

Comment:
Not necessary to fix before submitting. But I worry that since alternative names are optional this could produce a lot of extra logging and slow down the pipeline.

Reply:
Removed the logging